index.js

```diff
@@ -12,22 +12,26 @@ var utils = require('./lib/utils');
 // Woodlot logger middleware entry
 function middlewareLogger(config) {
 
-    // Check logger config for requried params
-    if (!config || !config.streams || !config.streams.length) {
-        // Log config warning to stdout 
-        utils.logConfigWarning();
-
-        // And continue express middleware chain execution
-        return function (req, res, next) {
-            next();
-        }
-    }
-
     // Sort config params for middleware execution
     var routeWhitelist = (config.routes && config.routes.whitelist) ? config.routes.whitelist : [];
     config.logToConsole = ('stdout' in config) ? config.stdout : true;
     config.logHeaders = (config.format && 'options' in config.format && 'headers' in config.format.options) ? config.format.options.headers : true;
 
+    // Check logger config for requried params
+    if (!config || !config.streams || !config.streams.length) {
+      if (!config.logToConsole) {
+          // Log config warning to stdout 
+          utils.logConfigWarning();
+
+          // And continue express middleware chain execution
+          return function (req, res, next) {
+              next();
+          }
+      }
+
+      config.streams ??= [];
```

Comment:
The logical nullish assignment doesn't work with Node JS versions below 15 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment#browser_compatibility
So we might have to alter this.

In fact since we are checking for the config value itself (for null, undefined or empty array) on [line 21](https://github.com/adpushup/woodlot/blob/4842e21494d3c01ed935cb57edbad067a2cdb3b0/index.js#L21), we can simply assign `config.streams = []` here.